world/quests/test/task-demo.lisp

```diff
@@ -0,0 +1,185 @@
+(:name find-android
+ :title "Find the Android"
+ :description "Locate the android rumoured to live in the mountain caves"
+ :invariant T
+ :condition quest:all-complete
+ :on-activate (fi-check pot-check tent-check sign-main-check sign-check landslide-check fire-check beacon-check corpse-check cave-in-check android-check home-check home-tent-check))
+(quest:interaction :name fi-check :interactable fi :dialogue "
+~ fi
+| (:annoyed)Next time I'll bring my own climbing gear.
+~ player
+| Suit yourself. But this was your idea.
+~ fi
+| Tracking the android was my idea. Talking to Harris was my idea. Clinging to your back while you scale a vertical cliff face - (:shocked)definitely not my idea.
+~ player
+| Well, we're here now. Can I proceed?
+~ fi
+| You know you don't need to ask me that, of all people? But yes, you are permitted.
+| Though I'm staying right here. Let me know what you find. And be careful.")
+(quest:interaction :name pot-check :interactable pot :dialogue "
+~ player
+| (The cooking pot infers human habitation.
+| (And this is a great spot for a lookout...)
```

Comment:
sounds good. I think I agree with the parentheticals being too distracting, especially since thoughts are going to be occurring in the field potentially more than speech
The suggested syntax would work for me +1